add_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import "testing"
+
+func TestAddComplexNumbers(t *testing.T) {
+    testCases := []struct {
+        name     string
+        a, b     complex128
+        expected complex128
+    }{
+        {"add zero", 0, 0, 0},
```

Comment:
Add another test scenario here, for large numbers (e.g. 9 digit imaginary component of the complex number).